src/app/beer_garden/api/stomp/server.py

```diff
@@ -0,0 +1,65 @@
+import stomp
+from brewtils.models import Event, Events, Request, Operation, System
+from brewtils.schema_parser import SchemaParser
+import beer_garden.events
+import beer_garden.router
+from beer_garden.api.stomp.processors import append_headers, process_event_message
+
+conn = None
+
+
+class OperationListener(stomp.ConnectionListener):
+    def on_error(self, headers, message):
+        print('received an error:', headers)
+
+    def on_message(self, headers, message):
+        global conn
+        operation = SchemaParser.parse_operation(
+            message, from_string=True
+        )
+        if hasattr(operation, 'kwargs'):
+            if 'wait_timeout' in operation.kwargs and operation.operation_type == "REQUEST_CREATE":
+                operation.kwargs['wait_timeout'] = 0
+        result = beer_garden.router.route(operation)
+        if result is not None:
```

Comment:
You need a way to handle exception and decide what to do when we throw an exception. Do we want to publish that out or just leave it in the logs?